src/generators/static/packageFileGenerator.ts

```diff
@@ -131,7 +135,7 @@ function regularAutorestPackage(
     scripts: {
       build:
         "tsc && rollup -c rollup.config.js && npm run minify && mkdirp ./review &&  npm run extract-api",
-      minify: `uglifyjs -c -m --comments --source-map "content='./dist/${packageDetails.nameWithoutScope}.js.map'" -o ./dist/${packageDetails.nameWithoutScope}.min.js ./dist/${packageDetails.nameWithoutScope}.js`,
+      minify: `uglifyjs -c -m --comments --source-map "content='./dist/index.js.map'" -o ./dist/${packageDetails.nameWithoutScope}.min.js ./dist/index.js`,
```

Comment:
Could you confirm this line of change with Will?

Reply:
This change accounts for the new name for the bundle (`index.js`) and here it uses that name for minifying. I forgot to update the value of the `main` entry in `package.json` to use the new name as well. I will push that change shortly after I regenerate smoke tests.

As for the reasoning to change the bundle name, I am just using what we currently do in Track 2 dataplane, I do not think it affects anything. cc @witemple-msft